README.md

```diff
@@ -90,74 +113,90 @@ If you want to run background jobs, which are necessary for spreadsheet bulk upl
 docker compose run web sidekiq start
 ```
 
-Note, if you update the Gemfile or Gemfile.lock, you will need to rebuild the web docker container.
+## Recommended Local Development
+
+Be sure all of the docker containers for dependent services are running in the background (e.g. solr, DSA) and stop the web container:
 
 ```
-docker compose build web
+docker compose up -d
+docker compose stop web
 ```
 
-### Debugging
+Start the development server - this should give you the Argo app on port 3000 mocking an admin login:
+
+```
+bin/dev
+```
 
-It can be useful when debugging to see the local rails server output in realtime and pause with 'byebug'.  You can do
-this while running in the app in the web container.  First stop any existing web container (if running already):
+Most of the time (unless you already have data), you will want seed data and a single item.  Note tha all servers must be running first, including web, and this will clear solr:
 
 ```
-docker compose stop web
+rake argo:seed_data
```

Comment:
You probably want `bin/rake` here so it's in the bundle context

Reply:
fixed